client/devdocs/docs-example/style.scss

```diff
@@ -0,0 +1,65 @@
+.docs-example__header {
+	display: flex;
+	align-items: center;
+}
+
+.docs-example__title {
+	flex: 1;
+}
+
+.docs-example__title,
+.docs-example__toggle {
+	margin: 0 !important;
+}
+
+.docs-example__header + .docs-example__main {
+	margin-top: 16px;
+}
+
+.docs-example__footer {
+	margin: 0;
+}
+
+.docs-example__stats {
+	border-bottom: 1px solid transparentize( lighten( $gray, 20% ), .5 );
+	font-size: 14px;
+	padding-bottom: 16px;
+}
+
+.docs-example__stats__item {
```

Comment:
>These classes don't follow our conventions.

I imagine that this is not flagged by our `wpcalypso/jsx-classname-namespace` rule because it [simply checks it is prefixed by and not equal to the folder name plus `__`](https://github.com/Automattic/eslint-plugin-wpcalypso/blob/2eacc852afa97e8489ff62a45541b858ae912aba/lib/rules/jsx-classname-namespace.js#L272-L277). Would it be accurate to say that the rest of the class name should consist of letters, numbers, and dashes only?